app/models/gobierto_common/scope.rb

```diff
@@ -3,14 +3,34 @@
 module GobiertoCommon
   class Scope < ApplicationRecord
     include GobiertoCommon::Sortable
+    include GobiertoCommon::Sluggable
+    include GobiertoCommon::ActsAsCollectionContainer
+    include User::Subscribable
 
     translates :name, :description
 
     belongs_to :site
     has_many :processes, class_name: "GobiertoParticipation::Process", dependent: :restrict_with_error
 
     validates :site, presence: true
+    validates :slug, uniqueness: { scope: :site }
 
     scope :sorted, -> { order(position: :asc, created_at: :desc) }
+
+    def self.alphabetically_sorted
+      all.sort_by(&:name)
```

Comment:
This method is only used in the tests? Could we remove it? The same happens in the `Issue class`.